Documentation/cmdref/index.rst

```diff
@@ -13,7 +13,6 @@ Command Reference
       :maxdepth: 0
       :titlesonly:
 
-      index_cilium_cli
```

Comment:
Out of curiosity, what's the rationale for this? The Cilium CLI is not part of the older, stable branches, I guess?

Reply:
Correct. We don't maintain multiple branches of the CLI. Rather than have stale unmaintained code in the older branches, we've opted to remove that code during branching.

It didn't occur to me that this means that the CLI reference here is largely useless though, because it'll only be generated based on `main` because that's the only place that has the actual code. We could argue if that's better than having outdated CLI docs on older branches of the Cilium docs, but I think this is just another sign that it's fundamentally wrong to have the CLI live inside of cilium/cilium :shrug:

Reply:
Probably if we want proper documentation of this kind for cilium-cli, it should live in cilium/cilium-cli and be pushed into a different docs website. Or otherwise we'd have to seriously overhaul the way we generate docs for docs.cilium.io to centralize branch-oblivious docs vs. the branch-specific docs.